platform/migration/platform-migration/src/test/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigratableTest.java

```diff
@@ -353,9 +355,18 @@ private void setup(String ddfHomeStr) throws IOException {
     System.setProperty(DDF_HOME_SYSTEM_PROP_KEY, ddfHome.toRealPath().toString());
     Path etcDir = ddfHome.resolve("etc");
     Files.createDirectory(etcDir);
+    setupBrandingFile(SUPPORTED_BRANDING);
     setupVersionFile(SUPPORTED_VERSION);
   }
 
+  private void setupBrandingFile(String branding) throws IOException {
+    final Path brandingFile = ddfHome.resolve("Branding.txt");
+
+    Files.createFile(brandingFile);
+    FileUtils.writeStringToFile(
```

Comment:
❓ Since `writeStringToFile` will create all needed directories + the file if needed, are we just using the `Files.createFile` for the IOException if the file already exists?

Reply:
writeStringFile() will create the file if it does not exist but not the directory structure.

Reply:
According to the javadoc, the directory structure **will* be created if missing.

https://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/FileUtils.html#writeStringToFile(java.io.File,%20java.lang.String,%20java.nio.charset.Charset)

Reply:
Might have been for the exception that Dan originally did that.